code/modules/religion/festival/instrument_rites.dm

```diff
@@ -0,0 +1,156 @@
+
+/**
+ * Perform the song effect.
+ *
+ * Arguments:
+ * * song_player - parent of the smooth_tunes component. This is limited to the compatible items of said component, which currently includes mobs and objects so we'll have to type appropriately.
+ * * song_datum - Datum song being played
+ */
+/datum/religion_rites/song_tuner/proc/song_effect(atom/song_player, datum/song/song_datum)
+	return
+
+/**
+ * When the song is long enough, it will have a special effect when it ends.
+ *
+ * If you want something that ALWAYS goes off regardless of song length, affix it to the Destroy proc. The rite is destroyed when smooth tunes is done.
+ *
+ * Arguments:
+ * * song_player - parent of the smooth_tunes component. This is limited to the compatible items of said component, which currently includes mobs and objects so we'll have to type appropriately.
+ * * song_datum - Datum song being played
+ */
+/datum/religion_rites/song_tuner/proc/finish_effect(atom/song_player, datum/song/song_datum)
+	return
+
+/datum/religion_rites/song_tuner/evangelism
+	name = "Evangelical Hymn"
+	desc = "Spreads the word of your god, gaining favor for each non-holy listener. At the end of the song, you'll bless all listeners, improving mood."
+	particles_path = /particles/musical_notes/holy
+	song_invocation_message = "You've prepared a holy song!"
+	song_start_message = span_notice("This music sounds blessed!")
+	glow_color = "#FEFFE0"
+	favor_cost = 0
+
+/datum/religion_rites/song_tuner/evangelism/song_effect(atom/song_player, datum/song/song_datum)
+	if(!song_datum || !GLOB.religious_sect)
+		return
+	for(var/mob/living/carbon/human/listener in song_datum.hearing_mobs)
+		if(listener == song_player || listener.anti_magic_check(magic = FALSE, holy = TRUE))
+			continue
+		if(listener.mind?.holy_role)
+			continue
+		if(!listener.ckey) //good requirement to have for favor, trust me
+			continue
+		GLOB.religious_sect.adjust_favor(0.2)
+
+/datum/religion_rites/song_tuner/evangelism/finish_effect(atom/song_player, datum/song/song_datum)
+	for(var/mob/living/carbon/human/listener in song_datum.hearing_mobs)
+		SEND_SIGNAL(listener, COMSIG_ADD_MOOD_EVENT, "blessing", /datum/mood_event/blessing)
+
+/datum/religion_rites/song_tuner/sooth
+	name = "Mending Chord"
+	desc = "Sing a sweet song, healing bruises and burns around you. At the end of a song, you'll seal one wound."
+	particles_path = /particles/musical_notes/heal
+	song_invocation_message = "You've prepared a healing song!"
+	song_start_message = span_nicegreen("This music is closing your wounds!")
+	glow_color = "#44FF84"
+	repeats_okay = FALSE
+
+/datum/religion_rites/song_tuner/sooth/song_effect(atom/song_player, datum/song/song_datum)
+	if(!song_datum)
+		return
+	for(var/mob/living/listener in song_datum.hearing_mobs)
+		if(listener.anti_magic_check(magic = FALSE, holy = TRUE))
+			continue
+		var/healy_juice = 0.6
+		if(listener.mind?.holy_role)
+			healy_juice*=3
+		listener.adjustBruteLoss(-healy_juice)
```

Comment:
is healing the only thing your brain is capable of coming up with as an effect?

we have an ENTIRE department for this.